styles/index.css

```diff
@@ -0,0 +1,70 @@
+h2 {
+    color: rgb(77, 53, 28);
+    font-size: 1.7rem;
+}
+
+h3 {
+    color: rgb(77, 53, 28);
+    font-size: 1.3rem;
+    margin-top: 10%;
+}
+.homepage-container {
+    display: grid;
+    grid-template: 0.1fr 0.4fr / 1fr;
+}
+.navbar-row {
+    grid-row: 2;
+    grid-column: 1/2;
+}
```

Comment:
Oh yes, I think this can be removed. Our original nav bar was in a different style, we forgot to remove some extra CSS code after we modify the nav bar.